src/views/scans/viewScansList.tsx

```diff
@@ -197,11 +197,12 @@ const ScansListView: React.FunctionComponent = () => {
                 icon={<EmptyStateIcon icon={PlusCircleIcon} />}
               />
               <EmptyStateBody>
-                Create a scan from the Sources page by selecting an individual source or multiple sources.
+                To create and initiate a scan, navigate to the Sources page. You can then choose one or more sources to
+                include in your scan.
```

Comment:
optional/future:
move strings into locale file since we already handle strings in this component